Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -402,106 +399,146 @@ namespace UnityEngine.InputSystem
+    ///     {
+    ///         var gamepad = Gamepad.current;
+    ///
+    ///         // No gamepad connected.
+    ///         if (gamepad == null)
+    ///         {
+    ///             return;
+    ///         }
+    ///
+    ///         // Check if "Button North" was pressed this frame
+    ///         if (gamepad.buttonNorth.wasPressedThisFrame)
+    ///         {
+    ///             Debug.Log("Button North was pressed");
+    ///         }
+    ///
+    ///         // Check if the button control is being continuously actuated and read its value
+    ///         if (gamepad.rightTrigger.IsActuated())
+    ///         {
+    ///             Debug.Log("Right trigger value: " + gamepad.rightTrigger.ReadValue());
+    ///         }
+    ///
+    ///         // Read left stick value and perform some code based on the value
+    ///         Vector2 move = gamepad.leftStick.ReadValue();
+    ///         {
+    ///             // Do 'move' code here
+    ///         }
+    ///
+    ///         // Creating haptic feedback while "Button South" is pressed and stopping it when released.
+    ///         if (gamepad.buttonSouth.wasPressedThisFrame)
+    ///         {
+    ///             gamepad.SetMotorSpeeds(0.2f, 1.0f);
+    ///
+    ///         } else if (gamepad.buttonSouth.wasReleasedThisFrame)
+    ///         {
+    ///             gamepad.ResetHaptics();
+    ///         }
+    ///     }
+    /// }
     /// </code>
     /// </example>
     /// </remarks>
+    /// <seealso cref="all"/>
+    /// <seealso cref="current"/>
+    /// <seealso cref="GamepadState"/>
+    /// <seealso cref="InputDevice"/>
+    /// <seealso cref="SetMotorSpeeds"/>
+    /// <seealso cref="ButtonControl.wasPressedThisFrame"/>
     [InputControlLayout(stateType = typeof(GamepadState), isGenericTypeOfDevice = true)]
     public class Gamepad : InputDevice, IDualMotorRumble
     {
         /// <summary>
         /// The left face button of the gamepad.
         /// </summary>
-        /// <value>Control representing the X/Square face button.</value>
         /// <remarks>
-        /// On an Xbox controller, this is the X button and on the PS4 controller, this is the
-        /// square button.
+        /// Control representing the X/Square face button.
+        /// On an Xbox controller, this is the <see cref="xButton"/> and on the PS4 controller, this is the
+        /// <see cref="squareButton"/>.
         /// </remarks>
-        /// <seealso cref="xButton"/>
-        /// <seealso cref="squareButton"/>
         public ButtonControl buttonWest { get; protected set; }
 
         /// <summary>
         /// The top face button of the gamepad.
         /// </summary>
-        /// <value>Control representing the Y/Triangle face button.</value>
         /// <remarks>
-        /// On an Xbox controller, this is the Y button and on the PS4 controller, this is the
-        /// triangle button.
+        /// Control representing the Y/Triangle face button.
```

Comment:
Side-note: I think this doc clearly illustrates the problem with current aliased enums. E.g. imagine if we supported 800 gamepads, how many variants of the mapping should be mentioned here? :)

Reply:
Yeah, but I think this only used for convenience. The aliases could (or maybe should)  be removed in a 2.x